hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/PartitionPerIndexStats.java

```diff
@@ -164,37 +165,37 @@ public long getMemoryCost() {
     }
 
     @Override
-    public void onInsert(long timestamp, IndexOperationStats operationStats, Index.OperationSource operationSource) {
+    public void onInsert(long timestampNanos, IndexOperationStats operationStats, Index.OperationSource operationSource) {
```

Comment:
not easier to call it startNanos? Timestamp doesn't add much on top of 'nanos' ; but already indicate time.